api/service/legacysync/syncing.go

```diff
@@ -1147,6 +1163,16 @@ type (
 	}
 )
 
+func ParseResult(res interface{}) (IsInSync bool, OtherHeight uint64, HeightDiff uint64) {
```

Comment:
I don't see any reason to be interface here.

Reply:
type changed